src/ontology/edits/oeo-shared-axioms.omn

```diff
@@ -2192,7 +1836,12 @@ pull request: https://github.com/OpenEnergyPlatform/ontology/pull/1219",
         OEO_00010121 some 
             (OEO_00000150 or OEO_00000331)
     
+    
+Class: OEO_00000323
 
+    DisjointWith: 
+        OEO_00000315
```

Comment:
In commit 985597298a6755167e6686f04550e071c5e40112, Protégé somehow reintroduces an disjoint axiom into oeo-shared. This seems to be related to a manual reinclusion of classes in commit 71345333d3c38ee09c8d0e04142d65bd7d33e864 of PR #1656.